feature-configs/e2e-gcp/performance/operator_catalogsource.patch.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: performance-addon-operators-catalogsource
+  name: performance-addon-operator-catalogsource
   namespace: openshift-marketplace
 spec:
-  image: quay.io/slintes/performance-addon-operators-registry
+  image: quay.io/slintes/performance-addon-operator-registry
```

Comment:
heads up, this image does not exist, I never pushed it yet. And if I did, it would contain changes from my work on my operator repo. So don't expect e2e on CI to work. I probably should never have committed my own image here, at least not without a dedicated tag. Sorry for spending confusion with this.

Reply:
@slintes @fromanirh umm, we need to address this. the performance addon should be promoting new images to quay after each merge occurs. I don't believe we're doing that right now.